api/carValue.js

```diff
@@ -0,0 +1,64 @@
+// Function for car value calculation
+const carValueCalculator = (carInfo) => {
+  console.log(carInfo);
+  const model = carInfo.model;
+  let year;
+
+  year = Number(carInfo.year);
+
+  if (!year) {
+    return "Error: invalid year - not a number";
+  }
+
+  const alphabet = [
```

Comment:
could turn this into a dict or map lookup ie ) const alphabet = {a:1,b:2,c:3, ...}. This means you can remove the 2nd loop and it will speed up the function as you arent looping 26 times for each char. Instead you just index in with char and get value.